public/docs/_examples/component-styles/ts/app/hero-controls.component.ts

```diff
@@ -17,10 +17,7 @@ import { Hero } from './hero';
 })
 // #enddocregion inlinestyles
 export class HeroControlsComponent {
-
   @Input() hero: Hero;
 
-  activate() {
-    this.hero.active = true;
-  }
+  activate() { this.hero.active = true; }
```

Comment:
Because it matches compactness used in other TS examples.
Anyhow, change undone.